### DISCUSSION TITLE
Inquiry Regarding Metric Naming in Reporting V3 API

### DISCUSSION THREAD

**Discussion by ivancda** [amzn/ads-advanced-tools-docs#208] [Q&A] [answered]
We are currently in the process of migrating to version 3 of the Reporting API (Campaign report types: SPONSORED_PRODUCTS, SPONSORED_BRANDS, and SPONSORED_DISPLAY) and have noticed a change in the naming of metrics between versions. We would like clarifications on whether metrics with different names in V3 are indeed equivalent in terms of collected data.

Examples of Naming Changes:

- attributedConversions14d (v2) -> purchases14d (SP) and purchasesClicks (SB and SD),
- attributedConversions14dSameSKU (v2) -> purchasesSameSku14d (SP) and purchasesPromotedClicks (SB and SD),
- attributedSales14d (v2) -> sales14d (SP) and salesClicks (SB and SD),
- attributedSales14dSameSKU (v2) -> attributedSalesSameSku14d (SP) and salesPromotedClicks (SB and SD),

Question:
Are the data from these metrics in the SP, SB, and SD Campaign report calls in V3 of the same nature? Can we aggregate this data into a single column in our application, or should we prepare distinct columns due to the potential difference in the nature of this data?

Best regards, thanks in advance.

**Comment by elizafarley** [comment, marked as answer]
Hi @ivancda, in version 3, only names have changed, not the source of the data. You can see all mappings in the [migration guide](https://advertising.amazon.com/API/docs/en-us/reference/migration-guides/reporting-v2-v3#metriccolumn-comparison).

**Reply by ivancda** [reply]
Hi @elizafarley , thank you very much for the reply, so I guess I can assume that data in the same row of this "migration guide" table is of the same nature and therefore can be aggregated, right? 

For example, if I want to show the user all the attributedConversions14d in SP, SB and SD Campaigns I can concatenate the values obtained from the columns "purchases14d" and "purchasesClicks" from their respective API calls.

Best regards

**Reply by elizafarley** [reply]
Yes if you combined attributedConversions14d in version 2 reporting across the three sponsored ads products, you could get the same value by combining purchases14d (for SP) and purchasesClicks (for SB and SD) for version 3 reporting.
